### PR TITLE
Create HelloWorld Project

### PR DESCRIPTION
**Pull request by miest-moore**
#1

### CODE REVIEW

File: TattooNationService/Controllers/WelcomeController.cs [hunk 1]
```diff
@@ -0,0 +1,21 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace TattooNationService.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class WelcomeController : ControllerBase
```
Comment (kmiestmoore) [inline TattooNationService/Controllers/WelcomeController.cs:7]: Should this be HelloWorldController?

Comment (miest-moore) [reply TattooNationService/Controllers/WelcomeController.cs:7]: No

Comment (kmiestmoore) [reply TattooNationService/Controllers/WelcomeController.cs:7]: Ok

**Review by kmiestmoore — APPROVED** [review]
Looks good

**Pull request merged**